.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -11,6 +11,8 @@ body:
           required: true
         - label: I can confirm the issues exists when I tested with `:latest`
```

```suggestion
        - label: I can confirm the issue exists when I tested with `:latest`
```

if you don't mind, there's an existing spelling mistake to fix up here

---

Done